model/indices/put_mapping/operations.smithy

```diff
@@ -4,12 +4,14 @@
 //  this file be licensed under the Apache-2.0 license or a
 //  compatible open source license.
 
+$version: "2"
 namespace OpenSearch
 
 @externalDocumentation(
     "OpenSearch Documentation": "https://opensearch.org/docs/latest/opensearch/rest-api/put-mapping/"
 )
 
+@idempotent
```

Comment:
Out of curiosity, what does this do? I'm having a hard time finding it in the docs.